### PR TITLE
Remove Print Statements / change password - notificaiton message

### PR DESCRIPTION
**Pull request by leong96**
Remove print statements, and shows a notification if, while changing the password, the current password is incorrect.

### CODE REVIEW

**Comment by leong96** [comment]
@eguy006 Double check whether you still need any of the removed print statements.

File: grasa_event_locator/views.py [hunk 1]
```diff
@@ -141,7 +142,7 @@ def changepw(request):
                             request.user.set_password(new)
                             request.user.save()
                     else:
-                            print("No")
+                            return()
```
Comment (justwheel) [inline grasa_event_locator/views.py]: `return` is a keyword, not a method. So this should be:

```suggestion
                            return
```

**Review by justwheel — APPROVED** [review]
Looks good @leong96. I am approving this, but please make the one change I suggested _before_ committing. Once you push up a new commit to change the `return` keyword, feel free to merge.

File: grasa_event_locator/views.py (hunk 1 above)
Comment (leong96) [reply grasa_event_locator/views.py]: Copy that! Suggestion committed.

**Pull request merged**